miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java

```diff
@@ -200,6 +216,13 @@ public ModelAndView setupForm(@PathVariable Long poolId, ModelMap model) throws
       model.put("owners", LimsSecurityUtils.getPotentialOwners(user, pool, securityManager.listAllUsers()));
       model.put("accessibleUsers", LimsSecurityUtils.getAccessibleUsers(user, pool, securityManager.listAllUsers()));
       model.put("accessibleGroups", LimsSecurityUtils.getAccessibleGroups(user, pool, securityManager.listAllGroups()));
+      Collection<SequencingParametersDto> sequencingParameters = Dtos.asSequencingParametersDtos(sequencingParametersService.getAll());
+      model.put("sequencingParameters", sequencingParameters);
+      JSONArray array = new JSONArray();
+      array.addAll(sequencingParameters);
+      model.put("sequencingParametersJson", array.toString());
+      model.put("platforms", getFilteredPlatforms(pool.getPlatformType()));
```

Comment:
Could use a `@ModelAttribute` method to include these in all models to avoid duplication